File: 1985-find-the-kth-largest-integer-in-the-array/1985-find-the-kth-largest-integer-in-the-array.js
/**
 * @param {string[]} nums
 * @param {number} k
 * @return {string}
 */
var kthLargestNumber = function(nums, k) {
   
    let arr=[]
    for(let i=0;i<nums.length;i++){
        let num=BigInt(nums[i])
        if (BigInt.asIntN(Number.MAX_SAFE_INTEGER, num) !== undefined) {
            arr.push(num);
        }
    }
     arr.sort((num1, num2) => {
        if (num1 === num2) {
            return 0;
        } else if (num1 > num2) {
            return -1;
        } else {
            return 1;
        }
    }); 
    return `${arr[k-1]}`
};

File: 1985-find-the-kth-largest-integer-in-the-array/1985-find-the-kth-largest-integer-in-the-array.py
class Solution:
    def kthLargestNumber(self, nums: List[str], k: int) -> str:
        arr=[]
        for i in range(len(nums)):
            arr.append(int(nums[i]))
        arr.sort()
        arr.reverse()
        return f'{arr[k-1]}'